File: 김형규/2019_01_08_code.js
function solution(clothes) {
    var answer = 1;
    const store = {};
    for(let i = 0; i < clothes.length; i++) {
        if(store[clothes[i][1]] === undefined)
            store[clothes[i][1]] = 1;
        else
            store[clothes[i][1]] += 1;
    }
    
    for(let i in store) {
        answer *= store[i]+1
    }
    return answer - 1;
}

File: 안남규/2019-01-08_code.py
import collections

def solution(clothes):
        answer=1
        x = []
        result = []
        for i in clothes:
                x.append(i[1])
                count = collections.Counter(x)
                y = list(count.values())
        for i in y:
                result.append(i+1)
        for i in range(len(result)):
                answer = answer * result[i]
        
        return answer-1